core/new-gui/src/app/common/service/workflow-persist/stub-workflow-persist.service.ts

```diff
@@ -76,4 +76,22 @@ export class StubWorkflowPersistService {
       return observer.next(results);
     });
   }
+  /**
+   * retrieves all workflow owners
+   */
+  public retrieveOwners(): Observable<string[]> {
+    const names = this.testWorkflows.filter(i => i).map(i => i.ownerName) as string[];
+    return new Observable(observer => {
+      observer.next([...new Set(names)]);
+    });
+  }
+
+  /**
+   * retrieves all workflow IDs
+   */
+  public retrieveIDs(): Observable<number[]> {
```

Comment:
rename it to `retrieveWorkflowIDs`?

Reply:
I was going to say that the name of the service is already workflow. But since all the other methods contain the workflow, I'll rename it just to be consistent